### ISSUE TITLE
nice but shaky

### ISSUE THREAD

**Issue by drarem** [BastiaanOlij/vehicle-demo#1]
The camera is vibrating or something, either it's orbiting the car when you first start, or the car is vibrating and turning around. Very strange, running 3.0.2 stable official,

**Comment by drarem** [comment]
ok seems to be the car itself shakes and rotates slightly if you don't move, when first starting.

**Comment by BastiaanOlij** [comment]
Yes, this is a known issue in the vehicle body simulation. I haven't had time to look further into it, hopefully Odino and I will have some time soon to figure out what can be done about it.

**Comment by vrai17** [comment]
this problem comes to 3.0.2 version
Ive tried this demo in 3.0.1dev version and this problem doesnt appear

**Comment by BastiaanOlij** [comment]
Hey @vrai17 , please check in 3.0.3 RC1, we did do some fixes in Godot itself that should improve some of the final shakiness we had.
Though for me it was working well in 3.0.2 after the fixes we applied to the project itself.
